tests/wpt/run.py

```diff
@@ -72,7 +73,10 @@ def set_defaults(kwargs):
         bin_name = "servo"
         if sys.platform == "win32":
             bin_name += ".exe"
-        bin_path = servo_path("target", bin_dir, bin_name)
+        if os.environ["CARGO_TARGET_DIR"]:
```

Comment:
Looks like we need `os.environ.get("CARGO_TARGET_DIR")` instead.

Reply:
My bad, didn't test without the variable :) should be fixed!